android/library/src/main/java/com/liferay/mobile/screens/ddm/form/service/BaseAPIOService.kt

```diff
@@ -15,21 +15,20 @@
 package com.liferay.mobile.screens.ddm.form.service
 
 import com.liferay.apio.consumer.ApioConsumer
-import com.liferay.apio.consumer.authenticator.BasicAuthenticator
-import com.liferay.mobile.screens.context.SessionContext
+import com.liferay.apio.consumer.configuration.AcceptLanguage
+import com.liferay.mobile.screens.util.ServiceUtil
+import java.util.*
 
 /**
  * @author Paulo Cruz
  */
-abstract class BaseAPIOService {
+abstract class BaseAPIOService(locale: Locale? = null) {
 
 	@JvmField
-	val apioConsumer: ApioConsumer
+	val apioConsumer: ApioConsumer = ApioConsumer(
+		*ServiceUtil.getAuthHeadersArray(),
```

Comment:
What is this `*` symbol for?

Reply:
It's the spread operator, using this the array is passed as a varargs parameter.

Reply:
oooh
I didn't know such a thing existed in kotlin
That is so coool

Reply:
hahaha, I liked that too